Samples~/Scripts/Playthrough/Metadata/AllowInteractableFunction.cs

```diff
@@ -9,16 +9,20 @@ internal class AllowInteractableFunction : MetadataFunction
 
         public override void Execute(string metadataValue)
         {
-            var entity = Dependencies.PlaythroughEntities.GetEntityByName(metadataValue);
+            string[] subjectTargets = metadataValue.Split(',');
```

Comment:
awesome, think we have this in some of our other PnP projects but I've forgot to port it over. Great call.